core/merginapi.h

```diff
@@ -191,6 +182,8 @@ class MerginApi: public QObject
     Q_PROPERTY( MerginSubscriptionInfo *subscriptionInfo READ subscriptionInfo NOTIFY subscriptionInfoChanged )
     Q_PROPERTY( QString apiRoot READ apiRoot WRITE setApiRoot NOTIFY apiRootChanged )
     Q_PROPERTY( bool apiSupportsSubscriptions READ apiSupportsSubscriptions NOTIFY apiSupportsSubscriptionsChanged )
+    // apiSupportsSelectiveSync if true, fetches mergin-config.json in project and changes sync behavior based on its content (selective sync)
+    Q_PROPERTY( bool apiSupportsSelectiveSync READ apiSupportsSelectiveSync NOTIFY apiSupportsSelectiveSyncChanged )
```

Comment:
may I suggest a different name for this? because the above "apiSupportsSubscriptions" is really about whether Mergin server has API for dealing with subscriptions, while selective sync does not deal with remote API. Maybe just drop "api" and have "supportsSelectiveSync" ?